source/contracts/Augur.sol

```diff
@@ -3,13 +3,98 @@ pragma solidity 0.4.17;
 
 import 'Controlled.sol';
 import 'libraries/token/ERC20.sol';
+import 'reporting/IUniverse.sol';
 
 
 // AUDIT/CONSIDER: Is it better that this contract provide generic functions that are limited to whitelisted callers or for it to have many specific functions which have more limited and specific validation?
 contract Augur is Controlled {
+    event MarketCreated(address universe, address indexed market, address indexed marketCreator, uint256 marketCreationFee, string extraInfo);
```

Comment:
`universe` should be indexed on all of these so the Augur Node can filter to only universes it cares about.

Reply:
Note, we are limited to 4 indexes per event, so this _may_ mean dropping one of the other indexes.  If that is necessary, we should discuss with @tinybike @epheph to figure out which one to drop.

Reply:
maximum 3 indexes per event, so it's even more critical

Reply:
Brought this up in Slack. I think we need @jack to weigh in.

Reply:
Brought this up in Slack too, but another option might be not sending the universe at all in these logs since they are sent to `Augur` by whitelisted contracts so we know they are legitimate and shouldn't need to filter them at all.

Reply:
Hm, currently it looks like just `market` and `marketCreator` are indexed, so you should be able to also add `universe` as an index.  Market and market creator aren't super important any more (b/c augur-node indexes markets already) but they're nice to have.

Reply:
The issue is specifically about the `Order` logs.

Reply:
Ah, gotcha -- I think we should index `universe`, `shareToken`, and `creator` for all the orders logs.

Reply:
That's a good point re: knowing they're legitimate.  Even so, I think it is still good to have universe as an index, so that if there are ever multiple (legitimate) universes, listeners can easily filter by universe address.